package.json

```diff
@@ -333,7 +333,7 @@
     "build-server": "./node_modules/.bin/gulp buildServer",
     "build-ext": "./node_modules/.bin/gulp buildExtension",
     "test-ui": "npm run test-compile && npm run test-ui-run",
-    "test-ui-run": "extest setup-and-run 'out/test/vscodeUiTest/suite/*.js' -u -c 1.42.1",
+    "test-ui-run": "extest setup-and-run 'out/test/vscodeUiTest/suite/*.js' -u",
```

Comment:
In master, the UI tests were being tested against VS Code version 1.42.1.

After this PR, the UI tests will always be tested against the latest version of VS Code. 

From README.md for https://github.com/redhat-developer/vscode-extension-tester:
> One aim of 3.x is to provide a more convenient way of updating to new VS Code releases. The modular approach allows us to publish updates to packages like page objects or locators without updating the main module. If such an update occurs, you should get it on the next npm install in your project without making changes to your package.json. Unless your package-lock.json decides otherwise.